opendc-model-odc/sc18/src/main/kotlin/com/atlarge/opendc/model/odc/topology/format/sc18/Sc18SetupParser.kt

```diff
@@ -74,7 +74,7 @@ class Sc18SetupParser(private val objectMapper: ObjectMapper = jacksonObjectMapp
                                     else -> throw IllegalArgumentException("The cpu id $id is not recognized")
                                 }
                             }
-                            InternalMachine(null, position, cpus.toSet(), emptySet())
+                            InternalMachine(null, position, cpus.toSet(), emptySet(), machine.ethernet_speed)
```

Comment:
Do you want camelCase in the setup JSON as well, or is this ok?

Reply:
No, that’s okay

Reply:
Never mind, the JSON parse still needs ethernet_speed, even if I change the field name.